Gemfile

```diff
@@ -1,117 +1,65 @@
 source 'https://rubygems.org'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+# required and confirmed in use
+# TODO: switch back to official gem when PR #36 is accepted and makes it into a release
+gem 'bento_search', git: 'https://github.com/magibney/bento_search', branch: 'search_controller_engine_params' #'1.7'
+gem 'blacklight', '6.8.0', git: 'https://github.com/magibney/blacklight.git', branch: 'v6.8.0-solr7-qq-feedFormat'
+gem 'blacklight-marc', '6.2.0'
+gem 'blacklight-ris', git: 'https://github.com/upenn-libraries/blacklight-ris.git'
+gem 'blacklight_advanced_search', '6.3.1'
+gem 'blacklight_alma', git: 'https://github.com/upenn-libraries/blacklight_alma.git'
+gem 'blacklight_solrplugins', git: 'https://github.com/upenn-libraries/blacklight_solrplugins.git'
+gem 'browserify-rails', '4.0.0'
+gem 'coffee-rails', '4.2.1' # no coffeescript in project...
+gem 'devise', '~> 4.6.0'
+gem 'devise-guests', '0.5.0'
+gem 'font-awesome-rails'
+gem 'httparty'
+gem 'jbuilder', '2.6.0'
+gem 'jquery-rails', '4.3.1'
+gem 'nokogiri', '~> 1.10.0'
+gem 'oga'
 gem 'rails', '~> 4.2.0'
-
-# we always need sqlite to the boundwiths database
-gem 'sqlite3', '1.3.12', platforms: :ruby
-
-group :test, :production do
-  gem 'mysql2', '~> 0.4.10'
-end
-
-gem 'jdbc-sqlite3', '3.8.11.2', platforms: :jruby
-
-gem 'activerecord-jdbcmysql-adapter', '1.3.21', platforms: :jruby
-
-# Use SCSS for stylesheets
+gem 'rsolr', '1.1.2'
 gem 'sass-rails', '5.0.6'
-
-# Use Uglifier as compressor for JavaScript assets
-gem 'uglifier', '3.0.2'
-
-# Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '4.2.1'
-
-# See https://github.com/rails/execjs#readme for more supported runtimes
+gem 'sdoc', '0.4.2', group: :doc
+gem 'sqlite3', '1.3.12', platforms: :ruby # we always need sqlite to the boundwiths database
+# TODO: switch back to official gem when PR #21 is accepted and makes it into a release
+gem 'summon', git: 'https://github.com/magibney/summon.rb', branch: 'sign-empty-param-values'
 gem 'therubyracer', '0.12.2', platforms: :ruby
+gem 'traject', '2.3.3'
+gem 'tzinfo-data', '1.2017.2' # newer passenger docker image needs tzinfo-data installed for some reason
+gem 'uglifier', '3.0.2'
 
-gem "blacklight", '6.8.0', :git => 'https://github.com/magibney/blacklight.git', :branch => 'v6.8.0-solr7-qq-feedFormat'
-
-gem "jettywrapper", '2.0.4'
-
-# newer passenger docker image needs tzinfo-data installed for some reason
-gem 'tzinfo-data', '1.2017.2'
-
-gem "solr_wrapper", '0.18.1'
-
-gem 'browserify-rails', '4.0.0'
-
-# Use jquery as the JavaScript library
-gem 'jquery-rails', '4.3.1'
-
+# investigate for removal
+gem 'blacklight_range_limit', '6.0.0'
```

Comment:
@magibney is this a relic from another time? I don't see this plugin used in any Franklin facets

Reply:
Yes, range-limit can be removed. Possibly interested in that at some point, but not used now, nor in the near future.